cmd/lncli/walletrpc_active.go

```diff
@@ -640,6 +647,13 @@ var fundPsbtCommand = cli.Command{
 			Usage: "(optional) the name of the account to use to " +
 				"create/fund the PSBT",
 		},
+		cli.StringFlag{
+			Name: "change_scope",
+			Usage: "(optional) the change scope to use to " +
+				"create/fund the PSBT. If no change scope " +
+				"is provided, then it will use the coin " +
+				"selection scope",
```

Comment:
should we also here add the remark that its p2tr for the default behaviour, bc I think the coin selection scope is understandable for a normal user, if you consider the "default" account its defaulted to p2tr although it has 3 scopes to choose from ?

Reply:
yes good point what do you think about:

`(optional) the change scope to use to create/fund the PSBT. If no change scope is provided, BIP0086 will be used for the default account and, otherwise, the scope of the provided account for any other not default accounts`

Is it clearer?

Reply:
will clarify the comment!

Reply:
logic changed